### PR TITLE
fix(mobile-booking-header): fix scroll issue

### PR DESCRIPTION
**Pull request by kantord**
fixes issue with scroll here:
https://github.com/kiwicom/smart-faq/issues/416#issuecomment-406720013<br/><br/><br/><url>LiveURL: https://smartfaq-fix-mobile-header-scrolling-issue.surge.sh</url>

### CODE REVIEW

File: src/common/layout/Header.js [hunk 1]
```diff
@@ -201,6 +201,15 @@ const renderLoggedOut = (
   );
 };
 
+type _SyntheticInputEvent = {
+  target: { // eslint-disable-line
```
Comment (jaroslav-kubicek) [inline src/common/layout/Header.js]: always opt to disable specific rule for line, not all

File: src/common/layout/Header.js [hunk 2]
```diff
@@ -211,10 +220,12 @@ class Header extends React.Component<Props, State> {
     window.addEventListener('scroll', this.handleScroll, true);
   }
 
-  handleScroll = ({ target }: SyntheticInputEvent<HTMLInputElement>) => {
+  handleScroll = ({ target }: _SyntheticInputEvent) => {
```
Comment (jaroslav-kubicek) [inline src/common/layout/Header.js]: does it work with `currentTarget`? https://github.com/facebook/flow/issues/2050#issuecomment-326773333

**Comment by jalbertsr** [comment]
It works very well in chrome dev tools, but trying it on my iPhone it works very buggy 🤷‍♂️

File: src/common/layout/Header.js (hunk 2 above)
Comment (kantord) [reply src/common/layout/Header.js]: No :'( For example `scrollTopMax` is missing

File: src/common/layout/Header.js (hunk 1 above)
Comment (kantord) [reply src/common/layout/Header.js]: :+1:

**Comment by kantord** [comment]
Should work now finally @jalbertsr

**Comment by kantord** [comment]
@jalbertsr how about now? :smile:

**Comment by kantord** [comment]
@jalbertsr how about now? :smile: :smile:

**Comment by jalbertsr** [comment]
I'm getting this error in sentry https://sentry.skypicker.com/kiwicom/smartfaqstaging/issues/1766290/
Any idea?

**Comment by kantord** [comment]
@jalbertsr No idea to be honest. I didn't touch anything related `performance.now` and I don't even now what it is :-D I'll look into this issue if it persists once the solution is working.

**Comment by kantord** [comment]
@jalbertsr now it works perfectly on my iPhone + browser. T'ho juro.

**Comment by kantord** [comment]
hmm I have to squash these

**Comment by kantord** [comment]
:heavy_check_mark: squashed

**Comment by kantord** [comment]
refactors pushed @jalbertsr @vborszeki

**Pull request merged**